ui/app/AppLayouts/Wallet/views/CollectiblesView.qml

```diff
@@ -160,13 +208,37 @@ ColumnLayout {
             }
             return AmountsArithmetic.toNumber(balance)
         }
+
+        property FunctionAggregator hasAllTimestampsAggregator: FunctionAggregator {
+            model: d.fullModel
+            initialValue: true
+            roleName: "lastTxTimestamp"
+
+            // aggregateFunction: (aggr, value) => aggr && !!value
+
+            aggregateFunction: function (aggr, value) {
+                console.info("bTest aggregateFunction", aggr, value);
+                return aggr && !!value;
+            }
+
+            onValueChanged: {
+                console.info("bTest onValueChanged", value)
+                d.toggleSortByDate(value);
```

Comment:
Does it mean that sorting will be changed automatically when timestamps are loaded? It's not intended behavior when user had explicitly selected other sorting, right?

Reply:
No, I enable the sort-by-date function, but toggleSortByDate maintains the current sort option.
I will rename it to  toggleSortByDateIsDisabled to be more clear